books/RayTracingTheRestOfYourLife.html

```diff
@@ -1193,10 +1248,36 @@
         std::cout << "I = " << sum/N << '\n';
     }
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    [Listing [integ-xsq-4]: <kbd>[integrate_x_sq.cc]</kbd> Integrating $x^2$, final version]
+    [Listing [integ-xsq-5]: <kbd>[integrate_x_sq.cc]</kbd> Integrating $x^2$, final version]
 </div>
 
-Which always returns the exact answer.
+This always returns the exact answer. Which, honestly, feels a bit like magic.
+
+
+A nonuniform PDF "steers" more samples to where the PDF is big, and fewer samples to where the PDF is
+small. By this sampling, we would expect less noise in the places where the PDF is big and more noise
+where the PDF is small. If we choose a PDF that is higher in the parts of the scene that have higher
+noise, and is smaller in the parts of the scene that have lower noise, we'll be able to reduce the
+total noise of the scene with fewer samples. This means that we will be able to converge to the
+correct scene _faster_ than with a uniform PDF. In effect, we are steering our samples toward the
+parts of the distribution that are more _important_. This is why using a carefully chosen nonuniform
+PDF is usually called _importance sampling_.
+
+In all of the examples given we always converged to the correct answer of $8/3$. We got the same
+answer when we used both a uniform PDF and the "correct" PDF ($i.e. f(d)=8d^{\frac{1}{3}}$). While
+they both converged to the same answer, the uniform PDF took much longer, after all, we only needed
+a single sample from the PDF that perfectly matched the integral. This should make sense, as we were
+choosing to sample the important parts of the distribution more often, whereas the uniform PDF just
+sampled the whole distribution equally, without taking importance into account.
+
+Indeed, this is the case for any PDF that you create. They
+will all converge eventually. This is just another part of the power of the Monte Carlo algorithm.
+Even the naive PDF above that split the distribution into two halves: 0 to $\sqrt{2}$ and $\sqrt{2}$
```

Comment:
“above that”? Seems like this could be deleted, but I'm not clear what you're trying to convey here.